packages/editor-ui/src/Interface.ts

```diff
@@ -936,49 +935,51 @@ export interface TargetItem {
 	outputIndex: number;
 }
 
-export interface IUiState {
-	sidebarMenuCollapsed: boolean;
-	modalStack: string[];
-	modals: {
-		[key: string]: IModalState;
-	};
+export interface NDVState {
```

Comment:
Nitpick: Should start with an _I_ (`INDVState`)  like other interfaces.

Reply:
I think this is something we should start to move away from..

Reply:
Ivan is big on getting rid of this